library/ssl_tls13_generic.c

```diff
@@ -552,6 +551,168 @@ int mbedtls_ssl_tls13_process_certificate( mbedtls_ssl_context *ssl )
     return( ret );
 }
 
+/*
+ *
+ * STATE HANDLING: Incoming Finished
+ * Overview
+ */
+
+static int ssl_tls13_preprocess_finished_in( mbedtls_ssl_context *ssl );
```

Comment:
```suggestion
static int ssl_tls13_prepare_finished_in( mbedtls_ssl_context *ssl )
{
    int ret;

    ret = mbedtls_ssl_tls1_3_calculate_expected_finished( ssl,
                    ssl->handshake->state_local.finished_in.digest,
                    sizeof( ssl->handshake->state_local.finished_in.digest ),
                    &ssl->handshake->state_local.finished_in.digest_len,
                    ssl->conf->endpoint ^ 1 );
    if( ret != 0 )
    {
        MBEDTLS_SSL_DEBUG_RET( 1, "mbedtls_ssl_tls1_3_calculate_expected_finished", ret );
        return( ret );
    }

    return( 0 );
}
```
To keep consistent name scheme

Reply:
Not addressed. Please move function body here.

Reply:
Address it.